samples/snapshot/pom.xml

```diff
@@ -23,14 +23,14 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-  <!-- {x-version-update-start::current} -->
   <dependencies>
+    <!-- {x-version-update-start:google-iam-admin:current} -->
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-iam-admin</artifactId>
```

Comment:
(Assuming this is the new artifact we recently talked about)

I was expecting this to have google-cloud-iam-admin as artifact ID. (”cloud” is missing) Is this intentional?

Reply:
It's in the `com.google.iam.admin` package rather than `com.google.cloud`, so yes it was intentional, but we could change it I suppose.

Reply:
If it's intentional, I'm good.